mhkit/river/io/d3d.py

```diff
@@ -6,6 +6,86 @@
 import numpy as np
 import pandas as pd
 import netCDF4
+import warnings
+
+
+def get_all_timestamps (data):
```

Comment:
I don't think what this returns is truly a timestamp. I think its really "seconds passed". I think we should call this something like `get_all_time` or maybe just `get_time`, or `get_seconds`. Then we need to work that into the return value as well. So maybe its now called 'time' or `seconds`.

Reply:
as discussed 

function `get_all_time`
returns `seconds_passed`